lib/vdsm/network/netswitch/configurator.py

```diff
@@ -205,7 +205,9 @@ def netinfo(vdsmnets=None, compatibility=None):
     ovs_nets, _ = util.split_switch_type(
         running_config.networks, running_config={}
     )
-    ovs_bonds = util.split_switch_type(running_config.bonds, running_config={})
+    ovs_bonds, _ = util.split_switch_type(
+        running_config.bonds, running_config={}
+    )
```

Comment:
Where is the typo? Is this a fix of another change? What is the issue this solves?

Reply:
The tuple was not properly assigned.

Reply:
Thanks, the commit message make the issue very clear now!